docs/user-guide/configuration.md

```diff
@@ -203,7 +203,7 @@ Or starting with `stylelint-config-standard`, then layering `myExtendableConfig`
 }
 ```
 
-The value of `"extends"` is a "locater" (or an array of "locaters") that is ultimately `require()`d, so can fit whatever format works with Node's `require.resolve()` algorithm. That means the a "locater" can be:
+The value of `"extends"` is a "locater" (or an array of "locaters") that is ultimately `require()`d, so can fit whatever format works with Node's `require.resolve()` algorithm. That means a "locater" can be:
```

Comment:
"the a" (two different articles) is never correct: it is one or the other. Looking at it in context, I think it could be either one, so this is correct.